scripts/i18n_utility/README.md

```diff
@@ -0,0 +1,106 @@
+# I18n Utility Script
+
```

Comment:
I think we could use an intro paragraph here that provides some larger context. It could cover:

1. How the translation files are created and map to the English files/directory structure
2. What happens when there's a mismatch between the translated directories and the English site content
3. What "I18n" means (I assume this is a funny shorthand for "internationalization" which, ick :D )
4. I'm sure there's something else I'm not considering